src/main/java/ToDos.java

```diff
@@ -0,0 +1,18 @@
+public class ToDos extends  Task{
+    private  String name;
+
+    public  ToDos(String name){
+        super(name);
+        this.name = name;
+
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString() ;
+    }
+}
+
+
+
```

Comment:
Perhaps you can delete these extra new lines?

Reply:
Nice mentioning that. I'm sorting out these empty lines